shell/browser/ui/electron_desktop_window_tree_host_linux.cc

```diff
@@ -99,9 +99,14 @@ void ElectronDesktopWindowTreeHostLinux::OnWindowTiledStateChanged(
   // of view.
   if (native_window_view_->has_frame() &&
       native_window_view_->has_client_frame()) {
-    static_cast<ClientFrameViewLinux*>(
-        native_window_view_->widget()->non_client_view()->frame_view())
-        ->set_tiled_edges(new_tiled_edges);
+    ClientFrameViewLinux* frame = static_cast<ClientFrameViewLinux*>(
+        native_window_view_->widget()->non_client_view()->frame_view());
+
+    bool maximized = new_tiled_edges.top && new_tiled_edges.left &&
+                     new_tiled_edges.bottom && new_tiled_edges.right;
```

Comment:
Q: is this same as `window_state_ == ui::PlatformWindowState::kMaximized` that it could be swapped with ?

Reply:
I see was not aware of the relevant upstream code. I am good with the current change then.